dotnetv3/StepFunctions/Scenarios/UIMethods.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+// snippet-start:[StepFunctions.dotnetv3.UIMethods]
+namespace StepFunctionsBasics;
+
+/// <summary>
+/// Some useful methods to make screen display easier.
+/// </summary>
+public class UiMethods
+{
+    private readonly string _sepBar = new('-', Console.WindowWidth);
+
+    /// <summary>
+    /// Show information about the scenario.
+    /// </summary>
+    public void DisplayOverview()
+    {
+        Console.Clear();
+        DisplayTitle("Welcome to the Amazon Step Functions Demo");
+
+        Console.WriteLine("This example application will do the following:");
+        Console.WriteLine("\t 1. Create an activity.");
+        Console.WriteLine("\t 2. Create a State Machine.");
+        Console.WriteLine("\t 3. Start an Execution.");
+        Console.WriteLine("\t 4. Run the worker, then stop it (sendTaskSuccess, sendTaskFailure)");
+        Console.WriteLine("\t 5. List Executions.");
+        Console.WriteLine("\t 6. Clean up the resources created for the example.");
+    }
+
+    /// <summary>
+    /// Display a message and wait until the user presses enter.
+    /// </summary>
+    public void PressEnter()
+    {
+        Console.Write("\nPlease press <Enter> to continue.");
+        _ = Console.ReadLine();
+    }
+
+    /// <summary>
+    /// Pad a string with spaces to center it on the console display.
+    /// </summary>
+    /// <param name="strToCenter"></param>
+    /// <returns></returns>
+    private string CenterString(string strToCenter)
+    {
+        var padAmount = (Console.WindowWidth - strToCenter.Length) / 2;
+        var leftPad = new string(' ', padAmount);
+        return $"{leftPad}{strToCenter}";
+    }
+
+    /// <summary>
+    /// Display a line of hyphens, the centered text of the title and another
```

Comment:
title and another --> title, and another

Reply:
Fixed